agent/app/agent.go

```diff
@@ -435,6 +435,17 @@ func (agent *ecsAgent) getEC2InstanceID() string {
 	return instanceID
 }
 
+// getoutpostARN gets the Outpost ARN from the metadata service
+func (agent *ecsAgent) getoutpostARN() string {
+	outpostARN, err := agent.ec2MetadataClient.OutpostARN()
+	if err != nil {
+		seelog.Warnf(
+			"Unable to access EC2 Metadata service to determine Outpost ID: %v", err)
+		return ""
```

Comment:
1.would this also be the case for a non-outpost instance?

2. what happens when an actual outpost does not have an ARN in the backend?

Reply:
> 1.would this also be the case for a non-outpost instance?
Yes, this will happen for the non-outpost instance. There is another check which ensures that if outpost ARN is empty it will not be sent in RCI call. 
> 
>     1. what happens when an actual outpost does not have an ARN in the backend?
In that case, it will be treated like a non-outpost instance. Outpost ARN is how backend will know it's running on outpost.

Reply:
can we edit the warn message to something like "Unable to obtain Outpost ARN from EC2 Metadata"?  Current one makes it looks like EC2 metadata is down.

Reply:
Yes will do.